sources/Renderer/Bootstrap/FieldRenderer/BsLinkedSetFieldRenderer.php

```diff
@@ -192,7 +193,7 @@ public function Render()
 								"render": function(data, type, row)
 								{
 									var oCheckboxElem = $('{$sSelectionInputHtml}');
-									if(row.limited_access)
+                                    if(row.limited_access)
```

Comment:
It looks like your editor has wrong indentation settings configured..
```suggestion
									if(row.limited_access)
```